### PR TITLE
Fix the wrapping issue for flag on review component

### LINKED ISSUE #10184: 'Flag' label has wrapping issues on mobile view
### Describe the problem and steps to reproduce it:

1. Open AMO on mobile 
2. Navigate to All reviews page for an add-on
3. Tap on the 'Flag' option

### What happened?
The label is wrapped

### What did you expect to happen?
The Flag label should not be wrapped 

### Anything else we should know?
The issue is reproducible on all AMO servers, so its not new

R10jlphQQ1

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #10184, just had a discussion with Bob , that it might work too, will remove the code for test if it works for everyone

### CODE REVIEW

File: src/amo/components/AddonReviewCard/styles.scss
Comment (bobsilverberg) [inline src/amo/components/AddonReviewCard/styles.scss:83]: Because this class deals with content from the `FlagReviewMenu` component, it should be put in the scss file for that component, not in this scss file.

**Review by bobsilverberg — CHANGES_REQUESTED** [review]
This looks good, thanks for the fix @championshuttler! I just have a comment about moving the definition of the css class.

**Comment by bobsilverberg** [comment]
@championshuttler There are also some failing tests that need to be addressed.

File: src/amo/components/AddonReviewCard/styles.scss
Comment (championshuttler) [reply src/amo/components/AddonReviewCard/styles.scss:83]: Sure will move that

**Comment by championshuttler** [comment]
> @championshuttler There are also some failing tests that need to be addressed.

Yeah I was just waiting for the approval so that i could chim  the patch

**Comment by bobsilverberg** [comment]
> > @championshuttler There are also some failing tests that need to be addressed.
> 
> Yeah I was just waiting for the approval so that i could chim the patch

Well, you're not going to get approval with a failing test 😉 , but I understand what you mean.

File: tests/unit/amo/components/TestFlagReviewMenu.js
Comment (championshuttler) [inline tests/unit/amo/components/TestFlagReviewMenu.js:68]: We are passing now a same class for all the classes so i dont think having these tests make sense. WDYT Bob?

File: src/amo/components/FlagReviewMenu/styles.scss
Comment (championshuttler) [inline src/amo/components/FlagReviewMenu/styles.scss:1]: Had to create a new am i missing something?>

**Comment by championshuttler** [comment]
Annddd
image

File: src/amo/components/FlagReviewMenu/styles.scss
Comment (bobsilverberg) [reply src/amo/components/FlagReviewMenu/styles.scss:1]: We do need a new file, but you shouldn't have to import `amo/css/styles` as this file isn't using anything from that file.

File: tests/unit/amo/components/TestFlagReviewMenu.js
Comment (bobsilverberg) [reply tests/unit/amo/components/TestFlagReviewMenu.js:68]: I agree.

File: tests/unit/amo/components/TestTooltipMenu.js
Comment (bobsilverberg) [inline tests/unit/amo/components/TestTooltipMenu.js:32]: No changes were made to `TooltipMenu`, so there shouldn't be any changes to this test.

File: src/amo/components/FlagReviewMenu/index.js
Comment (bobsilverberg) [inline src/amo/components/FlagReviewMenu/index.js:127]: We have a naming convention for the classNames in our components, where they start with the component name, for example, `DropdownMenu-button` in the `DropdownMenu` component, so this should be something like `FlagReviewMenu-menu`.

**Review by bobsilverberg — CHANGES_REQUESTED** [review]
Looks good, thanks for the update @championshuttler! Just a few comments to address.

**Review by bobsilverberg — APPROVED** [review]
Looks good, nice work @championshuttler!

**Pull request merged**
